dds/DCPS/transport/framework/TransportInst.cpp

```diff
@@ -145,6 +145,16 @@ OpenDDS::DCPS::TransportInst::impl()
   return impl_;
 }
 
+OpenDDS::DCPS::TransportImpl_rch
+OpenDDS::DCPS::TransportInst::get_impl()
+{
+  ACE_GUARD_RETURN(ACE_SYNCH_MUTEX, g, lock_, TransportImpl_rch());
+  if (!shutting_down_) {
+    return impl_;
+  }
+  return TransportImpl_rch();
```

Comment:
```suggestion
  return impl_;
```
I don't think it matters if it's shutting down here. We're only check above to make sure we don't create a new one during shutdown.